static/js/containers/PostDetailSidebar_test.js

```diff
@@ -0,0 +1,100 @@
+// @flow
+import R from "ramda"
+import sinon from "sinon"
+import { assert } from "chai"
+
+import { actions } from "../actions"
+import PostDetailSidebar, {
+  PostDetailSidebar as InnerPostDetailSidebar
+} from "./PostDetailSidebar"
+import IntegrationTestHelper from "../util/integration_test_helper"
+import { makePost } from "../factories/posts"
+import { makePostResult } from "../factories/search"
+
+describe("PostDetailSidebar", () => {
+  let helper, render, post, relatedPostData
+
+  beforeEach(() => {
+    post = makePost()
+    relatedPostData = R.times(makePostResult, 4)
+    helper = new IntegrationTestHelper()
+    helper.getRelatedPostsStub.returns(Promise.resolve(relatedPostData))
```

Comment:
yeah, I'm kind of stumped on this one tbh, I've looked through and it seems like there's no reason why it shouldn't run. My only thought is that is has to do with using the HOC renderer, it doesn't use a full redux store (it's a mock store instead) and I'm not sure if it fully supports the redux-thunk middleware. But that's just sort of a guess, I'm really not sure why it's not working.